packages/ui/react-ui-theme/src/styles/fragments/focus.ts

```diff
@@ -2,14 +2,7 @@
 // Copyright 2022 DXOS.org
 //
 
-import { mx } from '../../util';
-
-// TODO(burdon): Add m-1/2 to make focusRing visible.
-
-export const focusRing = mx(
-  'focus:outline-none focus-visible:z-[1] focus-visible:hover:outline-none dark:focus-visible:hover:outline-none focus-visible:ring-2 focus-visible:ring-offset-0',
-  'focus-visible:ring-primary-350 focus-visible:ring-offset-white dark:focus-visible:ring-primary-450 dark:focus-visible:ring-offset-black',
-);
+export const focusRing = 'ch-focus-ring';
```

Comment:
remove `mt-[2px]` in MarkdownEditor.tsx and Section.tsx?
assume Message.tsx is okay with it's current margin?

Reply:
That would creep scope in a way this PR probably shouldn’t attempt to resolve since here it’s just deferring to the utility classname for the same styles. I’d prefer to land this and follow-up on that design debt separately.

Reply:
ok -- you'd just asked me to remind you about the places I added margins -- as long as you have this in mind i'll merge this now.